.github/workflows/lint-test.yaml

```
@@ -61,15 +61,18 @@ jobs:
         if: steps.list-changed.outputs.changed == 'true' || github.event_name == 'workflow_dispatch'
 
       - name: Add repositories
-        run: helm dependency list . 2> /dev/null | tail +2 | head -n -1 | awk '$3 !~ /^file:/ { print "helm repo add " $1 " " $3 }' | while read cmd; do $cmd; done
+        run: |
+          for dir in $(ls -d charts/*/); do
+            helm dependency list $dir 2> /dev/null | tail +2 | head -n -1 | awk '$3 !~ /^file:/ { print "helm repo add " $1 " " $3 }' | while read cmd; do $cmd; done
+          done
 
       - name: Remove deployment failure triggers
         run: |
           # Remove <path:... directives
-          sed -Ei 's/<path:factory-x-ci-cd\/data\/async-aas\#([a-zA-Z0-9-]+)>/\1/g' values.yaml
+          sed -Ei 's/<path:factory-x-ci-cd\/data\/async-aas\#([a-zA-Z0-9-]+)>/\1/g' charts/async-aas-helm/values.yaml
           # Disable git init phase, we don't need initial model for testing the FA³ST deployment
-          sed -Ez 's/seeding:\n( *)enabled: true/seeding:\n\1enabled: false/' -i values.yaml
+          sed -Ez 's/seeding:\n( *)enabled: true/seeding:\n\1enabled: false/' -i charts/async-aas-helm/values.yaml
 
       - name: Run chart-testing (install)
-        run: ct install --target-branch ${{ github.event.repository.default_branch }} --charts .
+        run: ct install --target-branch ${{ github.event.repository.default_branch }} --charts ./charts/async-aas-helm
```

```suggestion
        run: ct install --target-branch ${{ github.event.repository.default_branch }} --charts --skip-existing ./charts/async-aas-helm
```
should fix the "fails of multiple versions" issue

---

I think this should be in the release.yaml file, I just added a comment below with the correct position